File: Data Structures & Algorithms/single-number/submission-1.cpp
class Solution {
public:
    int singleNumber(vector<int>& nums) {
        int ans=0;
        for(int i=0;i<nums.size();i++){
            ans=ans^nums[i];
            //把數字進行XOR，相同的就會消掉，剩下最後一個=答案
        }
        return ans;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        count_map=Counter(nums)
        
        min_heap=[]

        for num,freq in count_map.items():
            heapq.heappush(min_heap,(freq,num))
            
            if len(min_heap)>k:
                heapq.heappop(min_heap)

        return [num for freq,num in min_heap]